mantle/kola/harness.go

```diff
@@ -212,7 +212,7 @@ var (
 		{
 			// https://github.com/coreos/fedora-coreos-config/pull/1797
 			desc:  "systemd generator failure",
-			match: regexp.MustCompile(`systemd\[[0-9]+\]: (.*) failed with exit status`),
+			match: regexp.MustCompile(`[systemd|sd-execu]\[[0-9]+\]: (.*) failed with exit status`),
```

Comment:
Shouldn't this be something like `(systemd|\(sd-execut)` instead?

But hmm, ISTM like this regex is too liberal for the given description. systemd prints the path of the generator that failed, and we know that generators must be in one of the paths specified in `systemd.generator(7)`. So an alternative approach is a regex like `.*/system-generators/.* failed with exit status`.

For completeness, we should probably catch the other cases in https://github.com/systemd/systemd/blob/dafb406383610cc631345815919fd8a77ee0c4dc/src/basic/process-util.c#L663-L679, e.g.: `.*/system-generators/.* (failed with exit status|terminated by signal|failed due to unknown reason)`.

Reply:
updated!